x-pack/plugins/siem/public/common/components/exceptions/__examples__/index.stories.tsx

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
+
+import { ExceptionItem } from '../viewer';
+import { ExceptionListItemSchema, Operator } from '../types';
+
+const getMockExceptionItem = (): ExceptionListItemSchema => ({
+  id: '[insert_uuid_here]',
+  item_id: 'item-id',
+  created_at: '2020-04-23T00:19:13.289Z',
+  created_by: 'user_name',
+  list_id: 'test-exception',
+  tie_breaker_id: '77fd1909-6786-428a-a671-30229a719c1f',
+  updated_at: '2020-04-23T00:19:13.289Z',
+  updated_by: 'user_name',
+  name: '',
+  description: '',
+  comments: [],
+  tags: [],
+  _tags: [],
+  type: 'simple',
+  namespace_type: 'single',
+  entries: [
+    {
+      field: 'actingProcess.file.signer',
+      type: 'match',
+      operator: Operator.INCLUSION,
+      value: 'Elastic, N.V.',
+    },
+    {
+      field: 'host.name',
+      type: 'match',
+      operator: Operator.EXCLUSION,
+      value: 'Global Signer',
+    },
+  ],
+});
```

Comment:
You might be able to pull the mock items from the lists plugin since they live within the common section and be able to avoid adding another mock item here. The less mock data items we have, the less areas we have to update when we change structures.

Reply:
Yea, I should've added a to do there. This was just temporary until the structures are updated in the lists plugin. But def, will remove these then.